Kill THLogAdd

gchanan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #31254 Kill THTensor_(bhistc).
* #31218 Kill THSize.
* **#31217 Kill THLogAdd**
* #31216 Kill some unnecessary function declarations.

It doesn't seem to be used.

Differential Revision: [D18986642](https://our.internmc.facebook.com/intern/diff/D18986642)

gchanan committed:
Kill THLogAdd

It doesn't seem to be used.

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit 4e43dbc:
* **4/4** failures introduced in this PR



## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=4e43dbcaea16aa4e3694fe7edb803b3394ce9583).

### 3 new failures recognized by patterns
The following build failures don't appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934059) caffe2_onnx_py2_gcc5_ubuntu16_04_build (1/3)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73643921))
```
Dec 12 23:47:41 [ 48%] Building CXX object third_party/onnx/CMakeFiles/onnx.dir/onnx/common/assertions.cc.o 
Dec 12 23:47:41 [ 48%] Building CXX object third_party/onnx/CMakeFiles/onnx.dir/onnx/common/model_helpers.cc.o 
Dec 12 23:47:42 [ 48%] Linking CXX static library ../../lib/libonnx.a 
Dec 12 23:47:42 [ 48%] Built target onnx 
Dec 12 23:47:53 Scanning dependencies of target torch_cpu 
Dec 12 23:47:54 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/DynamicLibrary.cpp.o 
Dec 12 23:47:54 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/ThreadLocalDebugInfo.cpp.o 
Dec 12 23:47:54 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o 
Dec 12 23:47:54 [ 48%] Building CXX object caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/DLConvertor.cpp.o 
Dec 12 23:47:54 In file included from /var/lib/jenkins/workspace/aten/src/ATen/Context.cpp:16:0: 
Dec 12 23:47:54 /var/lib/jenkins/workspace/aten/src/TH/TH.h:12:25: fatal error: TH/THLogAdd.h: No such file or directory 
Dec 12 23:47:54 compilation terminated. 
Dec 12 23:47:54 make[2]: *** [caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o] Error 1 
Dec 12 23:47:54 make[2]: *** Waiting for unfinished jobs.... 
Dec 12 23:47:54 caffe2/CMakeFiles/torch_cpu.dir/build.make:298: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/__/aten/src/ATen/Context.cpp.o' failed 
Dec 12 23:47:54 make[1]: *** [caffe2/CMakeFiles/torch_cpu.dir/all] Error 2 
Dec 12 23:47:54 CMakeFiles/Makefile2:5422: recipe for target 'caffe2/CMakeFiles/torch_cpu.dir/all' failed 
Dec 12 23:47:54 make: *** [all] Error 2 
Dec 12 23:47:54 Makefile:138: recipe for target 'all' failed 
Dec 12 23:47:54 Building wheel torch-1.4.0a0+4e43dbc 
Dec 12 23:47:54 -- Building version 1.4.0a0+4e43dbc 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934051) pytorch_linux_xenial_py2_7_9_build (2/3)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73644104))
```
Dec 12 23:46:13 +++ eval 'extract_trap_cmd ' 
Dec 12 23:46:13 ++++ extract_trap_cmd 
Dec 12 23:46:13 ++++ printf '%s\n' '' 
Dec 12 23:46:13 +++ printf '%s\n' cleanup 
Dec 12 23:46:13 ++ trap -- ' 
Dec 12 23:46:13 cleanup' EXIT 
Dec 12 23:46:13 ++ which sccache 
Dec 12 23:46:13 ++ sccache --stop-server 
Dec 12 23:46:13 Stopping sccache server... 
Dec 12 23:46:13 error: couldn't connect to server 
Dec 12 23:46:13 caused by: Connection refused (os error 111) 
Dec 12 23:46:13 ++ true 
Dec 12 23:46:13 ++ rm /var/lib/jenkins/sccache_error.log 
Dec 12 23:46:13 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Dec 12 23:46:13 ++ true 
Dec 12 23:46:13 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Dec 12 23:46:13 ++ SCCACHE_IDLE_TIMEOUT=1200 
Dec 12 23:46:13 ++ RUST_LOG=sccache::server=error 
Dec 12 23:46:13 ++ sccache --start-server 
Dec 12 23:46:13 Starting sccache server... 
Dec 12 23:46:13 ++ sccache --zero-stats 
```
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/3934159) pytorch_linux_xenial_py3_clang5_mobile_build (3/3)
**Step:** "Build" ([details](https://dr.pytorch.org/build-details.html?build_id=73644355))
```
Dec 12 23:47:42 +++ eval 'extract_trap_cmd ' 
Dec 12 23:47:42 ++++ extract_trap_cmd 
Dec 12 23:47:42 ++++ printf '%s\n' '' 
Dec 12 23:47:42 +++ printf '%s\n' cleanup 
Dec 12 23:47:42 ++ trap -- ' 
Dec 12 23:47:42 cleanup' EXIT 
Dec 12 23:47:42 ++ which sccache 
Dec 12 23:47:42 ++ sccache --stop-server 
Dec 12 23:47:42 Stopping sccache server... 
Dec 12 23:47:42 error: couldn't connect to server 
Dec 12 23:47:42 caused by: Connection refused (os error 111) 
Dec 12 23:47:42 ++ true 
Dec 12 23:47:42 ++ rm /var/lib/jenkins/sccache_error.log 
Dec 12 23:47:42 rm: cannot remove '/var/lib/jenkins/sccache_error.log': No such file or directory 
Dec 12 23:47:42 ++ true 
Dec 12 23:47:42 ++ SCCACHE_ERROR_LOG=/var/lib/jenkins/sccache_error.log 
Dec 12 23:47:42 ++ SCCACHE_IDLE_TIMEOUT=1200 
Dec 12 23:47:42 ++ RUST_LOG=sccache::server=error 
Dec 12 23:47:42 ++ sccache --start-server 
Dec 12 23:47:42 Starting sccache server... 
Dec 12 23:47:42 ++ sccache --zero-stats 
```


---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

gchanan committed:
Update on "Kill THLogAdd"


It doesn't seem to be used.

Differential Revision: [D18986642](https://our.internmc.facebook.com/intern/diff/D18986642)

[ghstack-poisoned]
